lib/trpc/routers/project.ts

```
@@ -0,0 +1,430 @@
+
+export const projectRouter = router({
+  // Get all projects for current user
+  list: protectedProcedure
+    .input(
+      z.object({
+        status: z.enum(['ACTIVE', 'ARCHIVED', 'COMPLETED', 'ON_HOLD']).optional(),
+        visibility: z.enum(['PUBLIC', 'PRIVATE', 'TEAM']).optional(),
+      })
+    )
+    .query(async ({ ctx, input }) => {
+      const userId = ctx.session.user.id!!;
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)